Github/Confusing Things in Git Terminology.md

```diff
@@ -0,0 +1,64 @@
+## Confusing Terms in the Git Terminology
+
+Git and GitHub are at the core of open source in today's scenario. However, there are a lot of confusing terms that often seem similar yet can have conflicting meanings and uses. Let's eradicate these confusing terms by decoding their underlying meanings and sense.
+
+- ### Origin and Upstream
+
+From Git Documentation:
+> When a repo is cloned, it has a default remote called origin that points to your fork on GitHub, not the original repo it was forked from.
```

Comment:
Mention something along the lines of "when you fork a repo and clone the fork locally then their is a default remote"